src/components/toast-notifications/toast-notifications.html

```diff
@@ -0,0 +1,19 @@
+<div class="toast-notifications-list-pf">
+  <div ng-repeat="(alertID, alert) in (alerts | filterCollection : filter) track by (alertID + (alert.message || alert.details))" ng-if="!alert.hidden"
+       ng-mouseenter="setHover(alert, true)" ng-mouseleave="setHover(alert, false)">
+    <div class="toast-pf alert {{alert.type | alertStatus}}" ng-class="{'alert-dismissable': !hideCloseButton}">
+      <button ng-if="!hideCloseButton" type="button" class="close" ng-click="close(alert)" aria-hidden="true">
+        <span class="pficon pficon-close"></span>
+      </button>
+      <span class="{{alert.type | alertIcon}}" aria-hidden="true"></span>
+      <span class="sr-only">{{alert.type}}</span>
+      <span class="'toast-alert-message" ng-if="alert.message" style="margin-right: 5px;">{{alert.message}}</span>
+      <span ng-if="alert.details">{{alert.details}}</span>
+      <span ng-repeat="link in alert.links">
+        <a ng-if="!link.href" href="" ng-click="onClick(alert, link)" role="button" ng-attr-target="{{link.target}}">{{link.label}}</a>
+        <a ng-if="link.href" href="{{link.href}}" ng-click="onClick(alert, link)" ng-attr-target="{{link.target}}">{{link.label}}</a>
```

Comment:
if you have an href whose content is dynamic you should use ng-href instead of href, this guarantees you don't have a point in time where href is set to the actual string `{{link.href}}`